bootstrap/tests/package.json

```diff
@@ -2,13 +2,13 @@
   "name": "tests",
   "description": "TJBot hardware tests",
   "version": "1.0.0",
-  "author": "Justin Weisz <jweisz@us.ibm.com>",
+  "author": "TJBot Team",
```

Comment:
If we're moving away from individual attribution, which is fine, then I recommend changing this to `TJBot <tjbot@us.ibm.com>`